fluentd/fluent.conf

```diff
@@ -135,18 +134,36 @@
     bearer_token_file /var/run/secrets/kubernetes.io/serviceaccount/token
     ca_file /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
   </filter>
+  <filter **kibana**>
+    type record_transformer
+    enable_ruby
+    <record>
+      log ${err || msg}
+    </record>
+    remove_keys req,res,msg,name,level,v,pid,err
+  </filter>
   <filter kubernetes.**>
     type flatten_hash
     separator _
   </filter>
+  <filter kubernetes.**>
+    type record_transformer
+    enable_ruby
+    <record>
+      hostname ${kubernetes_host}
+      message ${log}
+      @version 3.1
+      @timestamp ${time.to_datetime}
+    </record>
+    remove_keys log,time,stream
+  </filter>
 
   <match **default** **openshift**>
-    @type elasticsearch
+    @type elasticsearch_dynamic
     host "#{ENV['OPS_HOST']}"
     port "#{ENV['OPS_PORT']}"
     scheme https
-    logstash_format true
-    logstash_prefix operations
+    index_name operations.${Time.at(time).getutc.strftime(@logstash_dateformat)}
```

Comment:
Could it be `.operations` so it won't conflict with logs from project `operations`?